ToneMapper/TMGLFeatureManagerViewController.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015 Lightricks. All rights reserved.
+// Created by Barak Yoresh.
+
+#import <UIKit/UIKit.h>
+
+#import "TMFeature.h"
+
+@class TMGLEngine;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Generic Feature view controller enabling canceling and saving of feature parameters.
+@interface TMGLFeatureManagerViewController : UIViewController <TMFeatureDelegate>
```

Comment:
The fact that the VC implements the <code>TMFeatureDelegate</code> should be hidden. It is used internally and  you probably don't want users of the VC to use. Exposing such internal interfaces publicly will give hard time in future if you want to refactor this class.